gulpfile.js

```diff
@@ -12,5 +12,6 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('app.less');
+    mix.less('admin.less');
+    mix.sass('main.scss');
```

Comment:
yeah okay [here](https://github.com/DoSomething/longshot/tree/f075f0922ab9454a2df8daa7ccdd3e1508dfe421/gulp/tasks) is all the old gulp build stuff. The [old gulpfile](https://github.com/DoSomething/longshot/blob/f075f0922ab9454a2df8daa7ccdd3e1508dfe421/gulpfile.js) ran all the tasks in that folder. Should that whole gulp folder live in `resources/assets`?